ITGuru.FourWheels/ITGuru.FourWheels.Data/Interfaces/IDataLayer.cs

```diff
@@ -1,25 +1,21 @@
 ﻿using ITGuru.FourWheels.Data.DataModels;
-using System;
-using System.Collections.Generic;
-using System.ComponentModel;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 
 namespace ITGuru.FourWheels.Data
 {
     public interface IDataLayer
     {
         void GenerateList();
         List<Customer> GetAllCustomers();
+        List<Customer> GetAllDeletedCustomers();
         bool AddCustomer(Customer customer);
         bool UpdateCustomer(Customer customer);
         bool HardDeleteCustomer(Guid customerID);
         bool SoftDeleteCustomer(Guid customerID);
         List<Vehicle> GetAllVehicles();
+        List<Vehicle> GetAllDeletedVehicles();
```

Comment:
`GetAllVehicles (Include deleted = false)`

Reply:
Changes have been made       GetAllVehicles() and GetAllCustomers()     have been changed to    GetAllVeicles(bool IncludeDeleted = false) and GetAllCustomers(bool IncludeDeleted = false)